requirements.dag.txt

```diff
@@ -10,5 +10,6 @@ Werkzeug>=1.0.1
 tqdm==4.62.0
 traitlets==5.1.1
 toolz==0.11.1
-Flask-AppBuilder==4.3.1
-black==21.8b0
\ No newline at end of file
+Flask-AppBuilder==4.3.10
+black==21.8b0
```

Comment:
Maybe out of scope but I noticed it here... the release notes for Airflow 2.8.0 say:

> Switch from Black to Ruff formatter

i.e. we shouldn't need the dependency anymore

(Not sure why Airflow even uses a code formatter)

Reply:
https://github.com/elifesciences/data-hub-issues/issues/848